src/content/docs/en/pages/architectures/edge-application/content-delivery.mdx

```diff
@@ -1,9 +1,14 @@
 ---
 title: Boost Content Delivery Speed and Reliability at the Edge
-description: Use edge-native content to replace legacy CDNs and leverage the speed and reliability of the edge network while reducing costs.
-meta_tags: edge, applications, web applications, api, content delivery, cdn
-namespace: docs_arch_content_delivery
+description: Explore how Azion Edge Platform enhances content delivery with optimized
+  cache policies, dynamic content handling, and improved user experiences.
```

Comment:
Sugestão: Explore how Azion Edge Platform enhances content delivery with optimized cache policies and dynamic content handling, improving user experiences.